utam-core/src/main/java/utam/core/selenium/element/Rect.java

```diff
@@ -0,0 +1,23 @@
+package utam.core.selenium.element;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+
+public class Rect {
+
+    private final Dimension windowSize;
+    private Point windowPoint;
+
+    public Rect(Point windowPoint, Dimension windowSize) {
+        this.windowPoint = windowPoint;
+        this.windowSize = windowSize;
+    }
+
+    public Point getPoint() {
```

Comment:
This is going to expose the Selenium `Point` and `Dimension` data structures to the user, right? We want to avoid that. Maybe expose `getX`, `getY`, `getWidth`, and `getHeight` instead? Or if we want to have a point and dimension, we should create our own data structures independent of those.